### DISCUSSION TITLE
DietPi-Software | Wazuh

### DISCUSSION THREAD

**Discussion by AdamF100001** [MichaIng/DietPi#6515] [Software Request]
Hi All

Would it be possible to add Wazuh to DietPi?

Wazuh is a free and open source security platform that unifies XDR and SIEM protection for endpoints and cloud workloads.

Link https://wazuh.com/

Kind Regards 

Adam

**Comment by martin-braun** [comment]
This tool is pretty powerful to supervise many boxes. Chuck made a video about it: https://www.youtube.com/watch?v=3CaG2GI1kn0

**Reply by AdamF100001** [reply]
Will this tutorial work with dietpi?

I watched the video weeks ago and wasn't sure if it would work.

Thanks for the video link.

**Reply by martin-braun** [reply]
I don't know. I just shared this link for the maintainers to learn about Wazuh and how it can be used. I'm looking forward to install Wazuh on my Pi in the future.

Since [Wazuh supports ARM](https://github.com/wazuh/wazuh/issues/5354), it should be possible to get it going without interference of the DietPi team, but as always, a prepackaged Wazuh would make things easier.

**Reply by Joulinar** [reply]
There is a pretty good documentation. Look they have own apt repository to install Debian packages.

**Reply by MichaIng** [reply]
See here: https://documentation.wazuh.com/current/installation-guide/wazuh-server/step-by-step.html
Switch to "APT". But skip `apt-transport-https`, this is a dummy package, not needed since years.
